scripts/mk-ros.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+ros="$1"
+
+ARCH=${ARCH:?"ARCH not set"}
+VERSION=${VERSION:?"VERSION not set"}
+
+cd $(dirname $0)/..
+
+strip_bin=$(which strip)
+if [ "${ARCH}" == "arm" ]; then
+  export GOARM=6
+  export CC=/usr/bin/arm-linux-gnueabihf-gcc
+  export CGO_ENABLED=1
+  strip_bin=/usr/arm-linux-gnueabihf/bin/strip
+fi
+GOARCH=${ARCH} go build -tags netgo -installsuffix netgo -ldflags "-X github.com/rancher/os/config.VERSION=${VERSION} -linkmode external -extldflags -static" -o ${ros}
+${strip_bin} --strip-all ${ros}
```

Comment:
Why are you doing this? I suppose for saving disk space...

Just asking cause I've read that it could harm `go` binaries and I don't know if it's supported by the go team.

It could be hard to debug if something goes wrong...